docs/docs/user-guide/relabeling.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 6
+---
+
+# Re-labeling
+
+**Learn how to send your data to labeling in Encord Annotate**
+
+When examining your project data, you might come across labels that appear incorrect or are missing altogether.
+With Encord Active, you have the ability to highlight such data and seamlessly transfer it to Encord Annotate, at the dedicated labeling stage for your project.
+This empowers annotators to address any missing elements and enhance the overall quality of your labels, ensuring a more accurate and comprehensive data annotation process.
+
+## Steps to send data to the labeling stage
+
+0. **[Optional]** Use the [filter feature](../user-guide/filtering) from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section to choose the desired data. 
+1. Access the Action tab from the toolbox in the _Data Quality_ or _Label Quality_ _Explorer_ section.
+2. Click the _🖋 Re-label_ button.
+3. Verify the number of tasks ready for submission to the labeling stage is correct and press the _Confirm_ button.  
+4. **[Optional]** Navigate to the project in Encord Annotate and explore the task modifications in the _Summary_ tab.
+
+![action-tab-in-toolbox](../images/user-guide/action-tab-in-toolbox.png)
+
```

Comment:
Would be nice with a short text and a screendump of what to expect in the Annotate platform after clicking the button.

Reply:
haha, yes, just waiting to build the confirmation button first 🦺 and add the image later